website/docs/r/policy_context_profile.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: "nsxt"
+page_title: "NSXT: nsxt_policy_context_profile"
+sidebar_current: "docs-nsxt-resource-policy-context-profile"
+description: A resource to configure a Context Profile.
+---
+
+# nsxt_policy_context_profile
+
+This resource provides a method for the management of a Context Profile.
+ 
+## Example Usage
+
+```hcl
+resource "nsxt_policy_context_profile" "test" {
+    display_name      = "test"
+    description       = "Terraform provisioned ContextProfile"
+    attributes {
+        data_type = "STRING"
```

Comment:
please correct the spacing in the example

Reply:
Sure, thanks for the catch. Will fix.